packages/patternfly-3/patternfly-react-extensions/src/components/CatalogTile/CatalogTile.js

```diff
@@ -23,7 +23,7 @@ const CatalogTile = ({
   const classes = classNames('catalog-tile-pf', { featured }, className);
 
   const defaultTruncateDescription = (text, max) => {
-    if (typeof text !== 'string' || text.length <= max) {
+    if (max === -1 || typeof text !== 'string' || text.length <= max) {
```

Comment:
ok... this just allows us to explicitly return text if desired (say you'd prefer to fade the visible characters if they overflow the title). I'm following now 😸